sky/clouds/gcp.py

```diff
@@ -492,6 +492,8 @@ def make_deploy_resources_variables(
             resources_vars['image_id'] = None
 
         resources_vars['disk_tier'] = GCP._get_disk_type(r.disk_tier)
+        if resources_vars['disk_tier'] == 'pd-extreme':
+            resources_vars['disk_iops'] = 20000
```

Comment:
Could we do this like the AWS implementation here? Reason: keeping related data in one function.

https://github.com/skypilot-org/skypilot/blob/40749e36e89c0b68420095083dacce7f3f56f774/sky/clouds/aws.py#L442

Reply:
Also, does `pd-{standard,balanced,ssd}` supports configure iops?

Reply:
Got it, let me wrap it with a function. Other disk types don't support configuration of iops, as mentioned in [GCP Disk Types](https://cloud.google.com/compute/docs/disks#disk-types).